main.js

```diff
@@ -68,5 +68,17 @@ define(function (require, exports, module) {
 
         // Register nohost server with highest priority
         LiveDevServerManager.registerServer({ create: _getServer }, 9001);
+
+        //Listen for paste event and run _configureLiveDev()
+        var myElement = document.getElementById("first-pane");
+        myElement.onpaste = function(e) {
+          if (window.clipboardData) { // IE
```

Comment:
These `if` blocks are unnecessary, since you aren't doing anything with `clipboardData`. Just 

```js
function(e) {
  _configureLiveDev();
  return true;
}
```

should be fine.